packages/ui/src/components/FloLegalDecisionTree/index.tsx

```diff
@@ -5,12 +5,20 @@ export interface FloLegalDecisionTreeProps extends DetailedHTMLProps<HTMLAttribu
   showOutcomes?: boolean;
   outcomesHeader?: string;
   outcomesHeaderLevel?: number;
+  dataUnansweredAlertMessage?: string;
+  dataUseRestartButton?: boolean;
+  dataPreviousButtonText?: string;
+  dataScrollOffset?: number;
 }
 export const FloLegalDecisionTree = ({
   encodedData,
   showOutcomes,
   outcomesHeader,
   outcomesHeaderLevel,
+  dataUnansweredAlertMessage,
+  dataUseRestartButton,
+  dataPreviousButtonText,
+  dataScrollOffset,
```

Comment:
Wij hoeven toch niet `data` in onze API op te nemen? Da's een implementatie-detail

```suggestion
  unansweredAlertMessage,
  UseRestartButton,
  previousButtonText,
  scrollOffset,
```